src/main/scala/is/hail/variant/Call.scala

```diff
@@ -0,0 +1,130 @@
+package is.hail.variant
+
+import java.io.Serializable
+
+import is.hail.check.Gen
+import is.hail.utils._
+import is.hail.variant.GenotypeType.GenotypeType
+
+object Call extends Serializable {
+
+  def apply(call: java.lang.Integer): Call = {
+    require(call == null || call >= 0, s"Call must be null or >= 0. Found ${call}.")
+    call
+  }
+
+  def toGenotype(call: Call): Genotype = {
```

Comment:
You can write `new Genotype(call)`.  Or `Genotype(call: Call)` can call `toGenotype`.  One or the other.